app/ydoc-server/src/ydoc.ts

```diff
@@ -38,6 +38,9 @@ export class WSSharedDoc {
   conns: Map<ConnectionId, Set<number>>
   awareness: Awareness
 
+  /**
+   * Create a {@link WSSharedDoc}.
+   */
```

Comment:
A nitpick, but our jsdoc lints allow making single-line docs, and I think it looks better for trivial explanations.
```suggestion
  /** Create a {@link WSSharedDoc} */
```
It's ok to go anyway, I may change such docs occasionally.